java/ql/lib/semmle/code/java/security/UnsafeDeserializationQuery.qll

```diff
@@ -519,3 +521,14 @@ private predicate joddJsonParserConfiguredUnsafely(Expr parserExpr) {
     not config.hasFlow(getASafelyConfiguredParser(), parser)
   )
 }
+
+/**
+ * Holds if `fromNode` to `toNode` is a dataflow step that stores data to a byte array field.
+ */
+private predicate byteArrayFieldFlowStep(DataFlow::Node fromNode, DataFlow::Node toNode) {
+  exists(FieldAccess access, Field field | field = access.getField() |
+    field.getType().hasName("byte[]") and
+    access = fromNode.asExpr() and
+    field.getAnAccess() = toNode.asExpr()
```

Comment:
Probably not an issue, but should this cast the access to `FieldRead` to explicitly exclude `FieldWrite`?
```suggestion
    field.getAnAccess().(FieldRead) = toNode.asExpr()
```

Reply:
Maybe. I'm going to move this to a separate PR, let's discuss it there. Thanks for the suggestion!